cmd/cerbos/repl/repl.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021-2022 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package repl
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/alecthomas/kong"
+	"github.com/cerbos/cerbos/cmd/cerbos/repl/internal"
+	"github.com/peterh/liner"
+)
+
+type Cmd struct {
+	History string `help:"Path to history file" type:"path"`
+}
+
+func (c *Cmd) Run(k *kong.Kong) error {
+	histFile := getHistoryFile(c.History)
+
+	reader := liner.NewLiner()
+	reader.SetMultiLineMode(true)
+
+	defer func() {
+		if err := writeHistory(reader, histFile); err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to persist history: %v", err)
+		}
+
+		_ = reader.Close()
+	}()
+
+	if err := loadHistory(reader, histFile); err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to read history: %v", err)
+	}
+
+	r, err := internal.NewREPL(reader, internal.NewPrinterOutput(k.Stdout, k.Stderr))
+	if err != nil {
+		return fmt.Errorf("failed to initialize the REPL: %w", err)
+	}
+
+	return r.Loop()
+}
+
+func getHistoryFile(path string) string {
+	if path == "" {
```

Comment:
In this case, perhaps we could put the file in `${XDG_DATA_HOME:~/.local/share}/cerbos/repl_history` to avoid polluting the user's home directory?